### ISSUE TITLE
NTR: THCA (trihydrocholestanoic acid) high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8253]
New term request THCA (trihydrocholestanoic acid) high in blood (segal_070920163940):   UMLS: 4229201

**Comment by pnrobinson** [comment]
See https://github.com/obophenotype/human-phenotype-ontology/issues/8836 for correction.

**Comment by pnrobinson** [comment]
see also https://github.com/obophenotype/human-phenotype-ontology/issues/8254

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000973
- label: Elevated circulating trihydroxycholestanoic acid concentration
- parents: [Abnormal circulating carbohydrate concentration (HP:0011013)](https://hpo.jax.org/app/browse/term/HP:0011013)
- pmid: 
- def: The concentration of trihydroxycholestanoic acid in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8253
- synonyms: Elevated circulating THCA concentration(NONE)
